modules/openy_features/openy_node/openy_node.install

```diff
@@ -5,6 +5,24 @@
  * Open Y Node install file.
  */
 
+/**
+ * Implements hook_install().
+ */
+function openy_node_install(){
+  $config = \Drupal::configFactory()->getEditable('views.view.frontpage');
+  $config->set('display.page_1.display_options.enabled', FALSE);
+  $config->save();
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function openy_node_uninstall() {
```

Comment:
Let's avoid uninstall guessing procedure...
No need to enable /node - we don't know

Reply:
I removed hook_uninstall().
Please, see again